sessionctx/stmtctx/stmtctx.go

```diff
@@ -95,6 +97,7 @@ type StatementContext struct {
 		warnings          []SQLWarn
 		histogramsNotLoad bool
 		execDetails       execdetails.ExecDetails
+		allExecDetails    []*execdetails.ExecDetails
```

Comment:
Should we reset it in `ResetForRetry`?

Reply:
Yes... you are right.
This problem has been fixed.